src/utils/parse-arguments-into-options.ts

```diff
@@ -74,7 +75,7 @@ export async function parseArgumentsIntoOptions(
       "-h": "--help",
     },
     {
-      argv: rawArgs.slice(2).map(a => a.toLowerCase()),
+      argv: rawArgs.slice(2),
```

Comment:
also, I removed lowerCasing here since uppercase letters are deprecated (see video) and it's not clear to user that name will be lowercased

Reply:
Noice yup!!